### DISCUSSION TITLE
(UPDATE)Smoke 'Em Out Struggle [FULL WEEK] | VS Garcello

### DISCUSSION THREAD

**Discussion by blueleavesclues** [luckydog7/Funkin-android#1678] [mods]
Complete Edition:
Link: [Complete Edition](https://drive.google.com/file/d/1oQGBLh8KXOJsdjTzXdPyfeJANYgEAhR2/view?usp=sharing)
Optimized Edition:
Link: [Optimized Edition](https://drive.google.com/file/d/1cYlznxx-VF-ed1lbNHHAh_9EbrhPz3-y/view?usp=sharing)

Differences from both version:
The Complete Edition has all Cutscenes & Dialog & Animations

The Optimized Edition Doesn't have anything but does contain Garcello's Animations

All of the images are from the Complete Edition
![20210713_093827](https://user-images.githubusercontent.com/80945262/125497593-1ab21144-43be-4080-8f89-6f49eb54c82d.jpg)
![20210713_093846](https://user-images.githubusercontent.com/80945262/125497596-e4fe5453-479b-42df-a7e8-11191911fddb.jpg)
![20210713_093906](https://user-images.githubusercontent.com/80945262/125497597-42496829-e752-4b99-87ff-d84db8761f92.jpg)
![20210713_094016](https://user-images.githubusercontent.com/80945262/125497601-41228986-e666-4cb1-a48d-a54661c70320.jpg)
![20210713_094040](https://user-images.githubusercontent.com/80945262/125497602-5c2ea860-e3ac-4046-a836-41f01d19e273.jpg)
![20210713_094103](https://user-images.githubusercontent.com/80945262/125497603-917bac80-7126-4010-b3a0-69d272625e4a.jpg)
![20210713_094112](https://user-images.githubusercontent.com/80945262/125497604-e755d01d-4787-4652-abcc-b575e251ada2.jpg)
![20210713_093801](https://user-images.githubusercontent.com/80945262/125497606-e6720299-b58b-43b7-819d-0db7390c0fe8.jpg)

**Comment by DzGithub2805** [comment]
Better than wyxos port
